classes/class-bsf-cm-auto-update.php

```diff
@@ -0,0 +1,81 @@
+<?php
+/**
+ * Plugin auto update class.
+ *
+ * @package     Contacts in Mautic
+ * @author      Brainstormforce
+ * @link        http://brainstormforce.com
+ * @since       1.0.3
+ */
+
+if ( ! class_exists( 'Bsf_CM_Auto_Update' ) ) :
+
+	/**
+	 * Bsf_CM_Auto_Update initial setup
+	 *
+	 * @since 1.0.3
+	 */
+	class Bsf_CM_Auto_Update {
+
+
+		/**
+		 * Class instance.
+		 *
+		 * @access private
+		 * @var $instance Class instance.
+		 */
+		private static $instance;
+
+		/**
+		 * Initiator
+		 */
+		public static function get_instance() {
+			if ( ! isset( self::$instance ) ) {
+				self::$instance = new self();
+			}
+			return self::$instance;
+		}
+
+		/**
+		 * Constructor
+		 */
+		public function __construct() {
+
+			// Plugin Updates.
+			add_action( 'init', __CLASS__ . '::init' );
```

Comment:
@nileshc-bsf Should this be called on `admin_init`?

Reply:
Action changes to "admin_init".